### PR TITLE
CONSOLE-3883: Update Console vs. dynamic plugin SDK package compat table

### PR DESCRIPTION
**Pull request by vojtechszocs**

### CODE REVIEW

File: frontend/packages/console-dynamic-plugin-sdk/README.md [hunk 1]
```diff
@@ -47,8 +47,14 @@ compatible versions of distributable SDK packages to versions of the OpenShift C
 
 | Console Version   | SDK Package                                     | Last Package Version |
 | ----------------- | ----------------------------------------------- | -------------------- |
-| 4.12.x            | `@openshift-console/dynamic-plugin-sdk`         | Latest               |
+| 4.15.x            | `@openshift-console/dynamic-plugin-sdk`         | Latest               |
```
Comment (jhadvig) [inline frontend/packages/console-dynamic-plugin-sdk/README.md:50]: Should we put `1.0.0` there directly, since we are going to publish it anyway

Comment (vojtechszocs) [reply frontend/packages/console-dynamic-plugin-sdk/README.md:50]: I was planning to do that post merge of #13566

**Review by jhadvig — APPROVED** [review]
/lgtm
/approve

**Comment by jhadvig** [comment]
Adding rest of the labels manually since we are only updating compat table

**Pull request merged**